app/javascript/components/pages/NotFound.js

```diff
@@ -5,8 +5,20 @@ class NotFound extends Component {
   render() {
     return (
     <>
-    <h3> OOPS, PAGE NOT FOUND!</h3> 
-    
+      <div className="notfound">
+        <img
+          className="notfound-image"
+          src="assets/desert2.jpeg"
+          alt="notfound"
+          width={"100%"}
+          height={"650px"}
+        >
```

Comment:
Just a suggestion, background: url() in CSS may have a similar approach and may simplify your code a little bit.